tests/fsharpqa/testenv/bin/.gitignore

```diff
@@ -1,7 +1,8 @@
-FSharp.Compiler.*
-FSharp.Core.*
-HostedCompilerServer.*
-ILComparer.*
-diff.*
-x86
-AMD64
\ No newline at end of file
+/FSharp.Compiler.*
+/FSharp.Core.*
```

Comment:
The leading `/` restricts it to matches in that directory, rather than also in subdirectories; it's probably not terribly important in this particular case.